client/state/jetpack-connect/actions.js

```diff
@@ -63,493 +63,504 @@ const remoteAuthPath =
 const remoteInstallPath = '/wp-admin/plugin-install.php?tab=plugin-information&plugin=jetpack';
 const remoteActivatePath = '/wp-admin/plugins.php';
 
-export default {
-	confirmJetpackInstallStatus( status ) {
-		return dispatch => {
-			dispatch( {
-				type: JETPACK_CONNECT_CONFIRM_JETPACK_STATUS,
-				status: status,
-			} );
-		};
-	},
+export function confirmJetpackInstallStatus( status ) {
+	return dispatch => {
+		dispatch( {
+			type: JETPACK_CONNECT_CONFIRM_JETPACK_STATUS,
+			status: status,
+		} );
+	};
+}
```

Comment:
Hmm, there's absolutely no reason for this to be a thunk -- this can be rewritten as a perfectly fine synchronous action. 😕 

```js
export function confirmJetpackInstallStatus( status ) {
	return( {
		type: JETPACK_CONNECT_CONFIRM_JETPACK_STATUS,
		status,
	} );
}
```

Since it's not related to the goal of this PR, and there a test for it, I guess we should probably do it in a separate PR. Still hurts my eyes...

Reply:
Yep, there are many in this file. I'll follow up to clean that up 👍

Reply:
see #20324